src/data/chains/10/chain.ts

```diff
@@ -91,6 +91,18 @@ export const optimism: TChain = {
       },
       redstoneTokenId: "OP",
     },
+    {
+      code: "TEC",
+      icon: tecIcon,
+      address: "0x8fc7c1109c08904160d6ae36482b79814d45eb78",
+      decimals: 18,
+      canVote: true,
+      priceSource: {
+        chainId: 10,
+        address: "0x8fc7c1109c08904160d6ae36482b79814d45eb78",
+      },
+      redstoneTokenId: "TEC",
```

Comment:
It looks like TEC isn't lsited on redstone so we won't get USD price on checkout (not a blocker but worth calling out)